automated install/basic-install.sh

```diff
@@ -187,6 +187,37 @@ if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
     echo -e "${OVER}  ${TICK} ${COL_LIGHT_GREEN}Port 53 is not in use, proceeding with install${COL_NC}"
   fi
 }
+
+#creating silent port 53 check
+silent53check(){
+# Print info regarding probing for open port 53
+# Probe localhost via 127.0.0.1 for open port 53 and sent flags
+if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
+  # what process is using 53?
+  silentwho53="$(${SUDO} lsof -i :53 +c 0 | awk 'FNR==2{ print $1 }')"
+  # check running process and see if it's pihole-FTL
+    if [ "$silentwho53" = "dnsmasq" ]; then
+      # If dnsmasq is present, set the dnsmas-flag to true for future reference
+      # (after packages and dependencies are installed).
+      # dnsmasq will be disabled at end of install, prior to FTLDNS start.
+      dnsmasq_flag=true
+#    elif [ "$silentwho53" = "systemd-resolve" ]; then
+      # If systemd-resolved is present, set the systemd-resolved-flag to true for future reference
+      # (after packages and dependencies are installed).
+      # systemd-resolved will be disabled at end of install, prior to FTLDNS start.
+#      systemd_resolved_flag=true
+    else
+    # port 53 is used by something else, stop install
+    echo -e "${OVER}  ${EXCL} ${COL_LIGHT_YELLOW}WARNING: Port 53 (mandatory for FTLDNS) is already in use by ${COL_LIGHT_RED}$silentwho53${COL_NC}.
+      Since this will interfere with the functionality of FTLDNS, the installer cannot continue.
+      Please visit ${COL_LIGHT_CYAN}https://discourse.pi-hole.net/t/ftldns-pi-holes-own-dns-dhcp-server/${COL_NC}
+      in order to get help related to this issue.
+      ${COL_LIGHT_RED}Installer will now exit.${COL_NC}"
```

Comment:
This is a silent function, so it should not print anything. If dnsmasq was successfully disabled already, this will cause the installer to incorrectly fail.